README.md

```diff
@@ -1 +1,49 @@
 # OTP Builds
+
+This is a collection of community-maintained [Erlang/OTP](https://github.com/erlang/otp) binary builds.
+
+Supported operating systems:
+
+  * `macos`
+
+Supported architectures:
+
+  * `amd64`
+  * `arm64`
```

Comment:
Should this state that AMD is only for 13, and ARM is only for 14 (current GitHub reality?)

Reply:
Might not be important, then. I was just stating that the runner for 13 is AMD (from GitHub) and the one for 14 is ARM.